### DISCUSSION TITLE
Visual appeal of the model

### DISCUSSION THREAD

**Discussion by zjafarpishe** [google/model-viewer#4366] [Q&A]
Hello, I have a 3D model of a watch with a steel strap. When I bring the model to the library, everything is perfect in the 3D display section, but when it goes to the augmented reality section, the material of the steel handle is lost and its material condition is damaged.
Please help me to have a completely realistic model

**Comment by elalish** [comment]
We have no way to help when you delete the issue template and don't include any images, screen captures, or repro links. Help us help you!
